### PR TITLE
scroll to step three on completion

### LINKED ISSUE #490: Make page scroll reset to top of Step 3 when submitted

### PR DESCRIPTION
**Pull request by dlouise64**
# What has changed and why?

now scrolls to top of text for step 3.

closes #490

# How to test this change

- [ ] has automated tests
- [ ] at least one a reviewer ran the code
- [ ] tested in small screen
- [ ] tested in Chrome
- [x] tested in Firefox
- [ ] tested in IE
- [ ] tested in Safari
- [ ] i18n
- [ ] tested in screen reader/contrast <-- remove this if no UX change

### CODE REVIEW

File: src/components/FirstTimeApplicant/WizardFormSecondPage.js [hunk 1]
```diff
@@ -162,9 +162,10 @@ class WizardFormSecondPage extends React.Component {
       <Fragment>
         <div className="theme-main">
           <Head/>
-            <form onSubmit={handleSubmit(this.saveFormData)}>
+            <form id="jumpTo"onSubmit={handleSubmit(this.saveFormData)}>
```
Comment (Br3nda) [inline src/components/FirstTimeApplicant/WizardFormSecondPage.js]: There's a space missing before `onSubmit`

**Comment by Br3nda** [comment]
This pull request **introduces 1 alert** when merging 12677fd4e99bc35a57b67b22101e945d4af28a30 into a3246d7a2e56ce348f3da7ecbcad0a71e0e58626 - [view on LGTM.com](https://lgtm.com/projects/g/ServiceInnovationLab/pancake-frontend/rev/pr-94f121e85dec86112eb6ddf63ca5d717cdf35111)

**new alerts:**

* 1 for Unused variable

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by Br3nda** [comment]
@dlouise64 this change a number like "30,000" into 30000. It seems to me this code could have a unit test for doing so - that it removes the comma, and that it adds up correctly.

**Comment by dlouise64** [comment]
@Br3nda Which space are you referring to?

**Comment by Br3nda** [comment]
Dana: 
image

**Pull request merged**